### PR TITLE
zib2020 FunctionalOrMentalStatus

### PR DESCRIPTION
**Pull request by ArdonToonstra**

### CODE REVIEW

**Review by ArdonToonstra — COMMENTED** [review]
Totally forgot this: in the _driverInclude.xsl a reference is made to the MedicalDevice template which is not yet working.  This still needs to be tested with the MedicalDevice branch.

File: ada_2_fhir-r4/zibs2020/ada_instance/nl-core-FunctionalOrMentalStatus-01.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                        </product>
+                        <product_omschrijving value="Ergonomisch elektrisch scheerapparaat" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.13"/>
+                        <begin_datum value="2015-02-23" datatype="datetime" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.11"/>
+                    </medisch_hulpmiddel>
+                </referenties>
+                <status_naam code="284888003" codeSystem="2.16.840.1.113883.6.96" displayName="kan zich scheren" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.2"/>
+                <status_waarde code="284892005" codeSystem="2.16.840.1.113883.6.96" displayName="moeite met scheren" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.3"/>
+                <status_datum value="2015-02-23" datatype="datetime" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.6"/>
+                <toelichting value="Dit is een toelichting" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.4"/>
```
Comment (LuudSlagter) [inline ada_2_fhir-r4/zibs2020/ada_instance/nl-core-FunctionalOrMentalStatus-01.xml:24]: Could we include a more speaking comment, such as 'Door verminderde fijne motoriek niet mogelijk om het gezicht volledig glad te scheren'

**Review by LuudSlagter — COMMENTED** [review]
Approved without taking into account the medical device

File: ada_2_fhir-r4/zibs2020/ada_instance/nl-core-FunctionalOrMentalStatus-01.xml (hunk 1 above)
Comment (ArdonToonstra) [reply ada_2_fhir-r4/zibs2020/ada_instance/nl-core-FunctionalOrMentalStatus-01.xml]: Done!

**Pull request merged**